adafruit_bno055.py

```diff
@@ -139,37 +145,41 @@
 
 
 class _ScaledReadOnlyStruct(Struct):  # pylint: disable=too-few-public-methods
-    def __init__(self, register_address, struct_format, scale):
+    def __init__(self, register_address: int, struct_format: str, scale: float) -> None:
         super().__init__(register_address, struct_format)
         self.scale = scale
 
-    def __get__(self, obj, objtype=None):
+    def __get__(self, obj, objtype=None) -> Tuple[float, float, float]:
```

Comment:
do `obj` and `objtype` here have types that we could add?

Reply:
this is my "iffy" understanding of objects.....see my changes